packages/cypress-commands/src/commands/login.js

```diff
@@ -53,5 +53,14 @@ const loginDev = () => {
  * https://docs.cypress.io/guides/guides/web-security.html#One-Superdomain-per-Test
  */
 export const login = () => {
+    ;[
+        'DEPRICATION WARNING:',
+        'The `login` function will is an older version.',
+        'It is left in place to prevent breakage.',
+        'Please use either the `loginThroughForm` command or `autoLogin` setup.',
+    ].forEach(message => cy.log(message))
+
     return loginDev()
```

Comment:
This ties into my other comments. I think we should get rid of the warning and just keep the command. Ideally, we should just use the `loginBasicAuth` approach at all time (i.e. just implement the `login` command as `loginBasicAuth`). If one approach to logging in works reliably we should use that and keep things simple.

What are the reasons for keeping both approaches around?

Reply:
backward compatibility

Reply:
A deprecation warning with an intent to remove in a future version is considerate towards users of the lib.

Reply:
Sorry but I think we have a misunderstanding here. Maybe I am missing something, but this is the situation as I've understood it:
- The original `cy.login` did not work properly and people have started building there own.
- This has produced another way of logging in, through the UI. This new way of logging in would now be introduced into this lib as `fillAndSubmitLoginForm`. However, as both @paschalidi and I have tried suggesting: we probably don't need this new method of logging in.
- So my conclusion was that we just need one command that we can use to reliably login, and which is not dependent on a UI that is only present in platform-apps.

So assuming the reasoning above is correct we shouldn't deprecated `cy.login`, but just fix it instead.

That was basically the main point I was trying to make in several of my comments and my review summary as well, see https://github.com/dhis2/cli-utils-cypress/pull/101#pullrequestreview-537370270. But I guess it wasn't very clear.

If things still aren't clear or I'm missing the point why my suggestion is not viable, let's discuss in the meeting we have together tomorrow @Mohammer5

Reply:
Alright, I misunderstood something between here and there. I thought that both commands work and then it makes sense to deprecate the less preferred on.

Reply:
@HendrikThePendric and I talked about this. We agreed that we don't need to deprecate this command, it does virtually the same as the `enableAutoLogin` without the hooks, so we can just update this command, and use it inside the setup function